doc/develop/getting_started/index.rst

```diff
@@ -708,10 +702,35 @@ Here are some next steps for exploring Zephyr:
 * Discover :ref:`project-resources` for getting help from the Zephyr
   community
 
+.. _troubleshooting_installation:
+
+Troubleshooting installation
+****************************
+
+Here are some tips for fixing some issues related to the installation process.
+
+.. _toolchain_zephyr_sdk_update:
+
+Double check Zephyr SDK variables when updating
+===============================================
+
+When updating Zephyr SDK, check whether the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` or :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables are already set.
+
+* If the variables are not set, the latest compatible version of the Zephyr SDK will be selected by default.
+  Proceed to next step without making any changes.
+
+* If :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` is set, the corresponding toolchain will be selected at build time
+  instead of the Zephyr SDK. Either unset the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` environment variable or change its
+  value to ``zephyr`` to make sure the Zephyr SDK is selected.
+
+* If the :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variable is set, it will override the lookup location
+  for the selected toolchain. If you install the Zephyr SDK to one of the locations recommended below,
```

Comment:
Not sure I understand the meaning of this sentence:
>  If the :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variable is set, it will override the lookup location for the selected toolchain. 

So if I set `ZEPHYR_TOOLCHAIN_VARIANT` to `gnuarmemb`, then how will setting `ZEPHYR_SDK_INSTALL_DIR` impact the lookup of the `gnuarmemb` toolchain ?

Reply:
Summoning @glarsennordic  for this technical question :)

Reply:
My apologies, I'm not actually 100% sure whether `ZEPHYR_SDK_INSTALL_DIR` applies only when `ZEPHYR_TOOLCHAIN_VARIANT="zephyr"` or if it impacts the lookup location for all of the toolchains.

> the selected toolchain

was sloppy phrasing on my part. It may be accurate, or it may not be. Someone actually familiar with the build configuration variables needs to chime in, and I don't know who that is.

If it only affects the zephyr SDK specifically, than better phrasing would be:

```rst
* If the :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variable is set, it will override the lookup location
  for the Zephyr SDK. If you install the Zephyr SDK to one of the locations recommended below,
```

Do we have any kind of reference documentation specifically dedicated to these environment variables, their function, and their possible values? That would help quite a bit.

Reply:
Applied edits.

Reply:
I just discovered `toolchain` has been changed to `Zephyr SDK` making the sentence clearer in meaning 👍 .
No further comments.